content/webapp/test/fixtures/search/index.ts

```diff
@@ -0,0 +1,637 @@
+      highlightTourType: 'audio',
+      title: 'Jason and the Adventure of 254 audio highlight tour',
+    },
+    {
+      type: 'Exhibition highlight tour',
+      id: 'ZsdFlRIAACIAqKGB',
+      uid: 'hard-graft',
+      description:
+        'Explore the profound impact of physical work on health and the enduring fight for workers’ rights through our new major exhibition.',
+      highlightTourType: 'audio',
+      title: 'Hard Graft: Work, Health and Rights audio highlight tour',
+    },
+    {
+      type: 'Page',
+      id: 'Zmq95BEAACMAGTxS',
+      uid: 'hard-graft--work--health-and-rights',
+      title: 'Hard Graft: Work, Health and Rights',
+      description:
+        'In September 2024, Wellcome Collection will present ‘Hard Graft’ a free major exhibition exploring experiences of physical work and its impacts on health and the body.',
+      tags: ['press'],
+    },
+    {
+      type: 'Page',
+      id: 'Wuw2MSIAACtd3Stm',
+      uid: 'rawminds-summer-2017-filmmaking-project',
+      title: 'RawMinds Summer 2017 Filmmaking Project',
+      description: 'See the highlights from this outdoor filmmaking project.',
+      tags: ['what-we-do', 'from_drupal'],
+    },
+    {
+      type: 'Event',
+      id: 'Xhyc0xAAACUAWYsj',
+      uid: 'bsl-tour-of-play-well',
+      title: 'BSL Tour of Play Well',
+      description:
+        'Enjoy highlights from the ‘Play Well’ exhibition accompanied by British Sign Language (BSL) guide Nadia Nadarajah.',
+      format: 'Gallery tour',
+      times: {
+        start: '2020-02-27T18:00:00.000Z',
+        end: '2020-02-27T18:45:00.000Z',
+      },
+    },
+    {
+      type: 'Event',
+      id: 'XUl7DBEAACIAzkwj',
+      uid: 'bsl-tour-of-misbehaving-bodies',
+      title: 'BSL Tour of Misbehaving Bodies',
+      description:
+        'Enjoy highlights from the ‘Misbehaving Bodies’ exhibition accompanied by British Sign Language (BSL) guide Golda Dahan.',
+      format: 'Gallery tour',
+      times: {
+        start: '2019-10-17T17:00:00.000Z',
+        end: '2019-10-17T17:45:00.000Z',
+      },
+    },
+    {
+      type: 'Event',
+      id: 'WvWoACIAAAvIBvMH',
+      uid: 'british-sign-language-tour-of-teeth',
+      title: 'British Sign Language tour of Teeth',
+      description:
+        "In this access tour, you'll see highlights from the Teeth exhibition accompanied by a British Sign Language (BSL) guide.",
+      format: 'Gallery tour',
+      times: {
+        start: '2018-07-19T17:00:00.000Z',
+        end: '2018-07-19T17:45:00.000Z',
+      },
+    },
+  ],
+  pageSize: 10,
+  totalPages: 12,
+  totalResults: 120,
+  nextPage:
+    'https://api.wellcomecollection.org/content/v0/all?query=highlight&page=2',
+  prevPage: null,
+  _requestUrl: '',
+} as ContentResultsList<Addressable>;
```

Comment:
I'm tempted to push towards typing these instead of using `as`; we've had very deprecated "fixtures" in the past that weren't maintained as they weren't typed and I think we're now trying to avoid it. 
I do understand that we're mostly testing whether or not there are results, here, though, but then if we're happy with it, maybe we can simplify them even further?